sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityTests.java

```diff
@@ -78,11 +80,39 @@ public void createUserWithResponseNullContext() {
     }
 
     @Test
-    public void createUserAndToken() {
+    public void createUserAndTokenWithChatScope() {
```

Comment:
The same comment that I left for `CommunicationIdentityAsyncTests.createUserAndTokenWithChatScope`

Reply:
Done